src/Library/demos/Navigation Split View/main.blp

```diff
@@ -0,0 +1,66 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.Window {
+  width-request: 360;
+  height-request: 200;
+  default-width: 640;
+  default-height: 480;
+
+  Adw.Breakpoint {
+    condition ("max-width: 400sp")
+    setters {
+      split_view.collapsed: true;
+      button.visible: true;
+    }
+  }
+
+  content: Adw.NavigationSplitView split_view {
+    sidebar: Adw.NavigationPage {
+      title: "Sidebar";
+      child: Adw.ToolbarView {
+
+        [top]
+        Adw.HeaderBar {
+          decoration-layout: "icon:close";
```

Comment:
```suggestion
          decoration-layout: "icon:close";
```

This one's not in the original.

Reply:
I added it because the original demo only had a close button and I think Workbench adds its own window decoration. But I can remove it.

Reply:
Ah, I see. I double-checked just now and it seems okay without, but it was worth checking, good thought.